docs/introduction/Car-registry-tutorial.rst

```diff
@@ -169,25 +174,25 @@ This transaction allows us to buy a car or recall a car sell order.
             
         *Output boxes:*
         
-            Two possible outputs are possible. In the case of buying a car, new CarBox with a new owner, a new Regular box with a value declared in carBuyOrderInfo for the Car's former owner. 
+            Two possible outputs are possible. In the case of buying a car, new CarBox with a new owner, a new Regular box with a value declared in carBuyOrderInfo for the car's former owner. 
 
 Car registry implementation
 ###########################
 
 First of all, we need to define new boxes. 
-As described before, a Car Box is a non-coin box as defined before we need Car Box Data class to describe custom data. So we need to define CarBox and CarBoxData as separate classes for setting proper way to serialization/deserialization.  
+As described before, a Car Box is a non-coin box, and similarly we need Car Box Data class to describe custom data. So we need to define CarBox and CarBoxData as separate classes to allow proper serialization/deserialization.  
 
 Implementation of CarBoxData:
 *****************************
 
-CarBoxData is implemented according description from ``Custom Box Data Creation`` section as ``public class CarBoxData extends AbstractNoncedBoxData<PublicKey25519Proposition, CarBox, CarBoxData>`` with custom data as:
+CarBoxData is implemented according to the description from ``Custom Box Data Creation`` section as ``public class CarBoxData extends AbstractNoncedBoxData<PublicKey25519Proposition, CarBox, CarBoxData>`` with custom data as:
```

Comment:
@zennermac 
CarBoxData is implemented according to the description from the ``Custom Box Data Creation`` section as a ``public class CarBoxData extends AbstractNoncedBoxData<PublicKey25519Proposition, CarBox, CarBoxData>`` with custom data as: